src/nm-config.c

```diff
@@ -743,6 +743,8 @@ typedef struct {
 	bool is_connection:1;
 } ConfigGroup;
 
+/** START OPTION LIST **/
```

Comment:
Please comment that `check-config-options.sh uses this`, otherwise it wouldn't be clear why is it there and shouldn't be removed.